app/views/pages/confirmation.html.erb

```diff
@@ -0,0 +1,11 @@
+<section class="header-container">
+  <%= render partial: "app/#{ENV['APP_NAME']}/header_container" %>
+</section>
+
+<div style="height: 250px">
+  <h4 style="background-color: lightyellow; padding:0.5em 1em;">Organization update successfully submitted. An administrator will review your request.</h4>
+</div>
+
+<div>
+  <p>Please, click <%= link_to "here", root_path %> to go to the homepage!</p>
+</div>
```

Comment:
This is not necessary we have the link at the top of the page to go back to the home page. Let's try to keep it as minimal as possible

Reply:
Sure, I will remove it!